projects/packages/my-jetpack/src/products/class-videopress.php

```diff
@@ -156,4 +156,24 @@ public static function get_manage_url() {
 			return admin_url( 'admin.php?page=jetpack#/settings?term=videopress' );
 		}
 	}
+
+	/**
+	 * Checks whether the current plan (or purchases) of the site already supports the product
+	 *
+	 * @return boolean
+	 */
+	public static function has_required_plan() {
+		$purchases_data = Wpcom_Products::get_site_current_purchases();
+		if ( is_wp_error( $purchases_data ) ) {
+			return false;
+		}
+		if ( is_array( $purchases_data ) && ! empty( $purchases_data ) ) {
+			foreach ( $purchases_data as $purchase ) {
+				if ( 0 === strpos( $purchase->product_slug, static::get_wpcom_product_slug() ) ) {
```

Comment:
`jetpack_videopress` is not the only purchase slug that includes VP. A list can be found here: https://github.com/Automattic/jetpack/blob/trunk/projects/packages/videopress/src/client/admin/hooks/use-plan/index.ts#L53-L91